recipe/run_test.bat

```diff
@@ -0,0 +1,11 @@
+@echo on
+
+REM static libraries:
+if not exist %PREFIX%\lib\libgmp.a exit /b 1
+if not exist %PREFIX%\lib\libgmpxx.a exit /b 1
+REM dynamic libraries:
+if not exist %PREFIX%\bin\libgmp-10.dll exit /b 1
+if not exist %PREFIX%\bin\libgmpxx-4.dll exit /b 1
+REM headers:
+if not exist %PREFIX%\include\gmp.h exit /b 1
+if not exist %PREFIX%\include\gmpxx.h exit /b 1
```

Comment:
Did we stop doing the test compilation to multiply two numbers together?

Reply:
I drop it. I think we need to provide the import `.lib` for the compiler but we do not create it

Reply:
I'll try to use without `.lib`

Reply:
Fixed